resources/options.html

```diff
@@ -100,6 +100,36 @@
         </div>
     </div>
 
+    <!--TODO: Make text understandable to the user (will be done when backend is properly working.-->
+    <div class="col s12 m6">
+        <div class="card" id="donotwatch">
```

Comment:
Naming: shouldn't this ID be 'doNotWatch'?
Same holds for the IDs below, with the exception of the abbreviations.